### PR TITLE
support mgm_7b

### PR DESCRIPTION
**Pull request by TousenKaname**

### CODE REVIEW

File: vlmeval/config.py [hunk 1]
```diff
@@ -22,6 +23,7 @@
     'MiniCPM-V':partial(MiniCPM_V, model_path='openbmb/MiniCPM-V'),
     'MiniCPM-V-2':partial(MiniCPM_V, model_path='openbmb/MiniCPM-V-2'),
     'OmniLMM_12B':partial(OmniLMM12B, model_path='openbmb/OmniLMM-12B', root=OmniLMM_ROOT),
+    'MGM_7B':partial(Mini_Gemini, name='MGM_7B', root=Mini_Gemini_ROOT)
```
Comment (kennymckormick) [inline vlmeval/config.py:26]: name -> model_path

File: vlmeval/vlm/mgm.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+from .base import BaseModel
+from PIL import Image
+
+
+class Mini_Gemini(BaseModel):
+
+    INSTALL_REQ = True
+    INTERLEAVE = False
+
+    def __init__(self, name, root='/mnt/petrelfs/wangguoan/git_repo/MGM', conv_mode='llava_v1', temperature= 0, **kwargs):
```
Comment (kennymckormick) [inline vlmeval/vlm/mgm.py:15]: kwargs is other arguments used for generation

File: vlmeval/vlm/mgm.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        tokenizer, model, image_processor, context_len = load_pretrained_model(model_path, None, model_name)
+
+        self.model = model
+        self.tokenizer = tokenizer
+        self.image_processor = image_processor
+        self.conv_mode = conv_mode
+        self.temperature = temperature
+
+
+    def get_prompt_from_message(self, message):
```
Comment (kennymckormick) [inline vlmeval/vlm/mgm.py:51]: keep the first image

Comment (kennymckormick) [reply vlmeval/vlm/mgm.py:51]: use `message_to_promptimg`

**Review by junming-yang — APPROVED** [review]

**Pull request merged**